documentation/src/main/asciidoc/user_guide/user_guide.asciidoc

```diff
@@ -36,3 +36,4 @@ include::rolling_upgrades.adoc[]
 include::extending.adoc[]
 include::architecture.adoc[]
 include::counters.adoc[]
+include::multimapcache.adoc[]
```

Comment:
I think we should move both counters and multimapcache docs further up, after query, before CDI.